testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -584,6 +584,21 @@ Feature: Synchronize products in the products page of the Setup Wizard
     And I wait until I see "SUSE Liberty Linux 7 x86_64" product has been added
     And I wait until all synchronized channels for "res7" have finished
 
+@susemanager
+@centos7_minion
```

Comment:
Should we use a suse liberty linux tag ?

Reply:
From what I can see, we do not have a liberty tag (yet).

Reply:
I see. It's a bit confusing. Do I understand correctly : the purpose is to deploy centos7, sync suse liberty linux 7, bootstrap centos7 using this suse liberty 7 and then migrate centos7 to liberty linux ?

Reply:
Yes, sorry, my bad. Maybe I should have made the description more clearly. 
So this PR only adds SLL 7 LTSS to the test suite, basically. We will still onboard CentOS 7 with the old CentOS 7 AK and with https://github.com/SUSE/spacewalk/issues/24884 we add/adapt the feature where we do the CentOS 7 migration to SLL 7 LTSS.

Reply:
Maybe I'm wrong but we currently don't sync the product for centos 7  (RHEL x86_64 Server 7).
I can see in for add Mu and activation key for centos7 we are looking for the paren RHEL 7 but we don't sync this product in sync all product ( `    'centos7_minion' => 'RHEL x86_64 Server 7',`
)

Reply:
Ho no, I'm wrong. SUSE Liberty Linux 7 is RHEL x86_64 Server 7 ... Ok no worries !

Reply:
Yes. With syncing and onboarding CentOS 7 nothing changes.